src/Network/Network.Test/Network.Test.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="25.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="26.0.0" />
```

Comment:
all projs referred to this SDK need to be updated, see [here ](https://cs.github.com/Azure/azure-powershell?q=PackageReference+Include%3D%22Microsoft.Azure.Management.Network%22)to get their list

Reply:
Done